assets/src/stories-editor/components/story-block-drop-zone.js

```diff
@@ -88,30 +88,19 @@ class BlockDropZone extends Component {
 			[ leftPosKey ]: getPercentageFromPixels( 'x', clonePosition.left - wrapperPosition.left ),
 			[ topPosKey ]: getPercentageFromPixels( 'y', clonePosition.top - wrapperPosition.top, baseHeight ),
 		} );
-	}
-
-	render() {
-		return (
-			<DropZone
-				className="editor-block-drop-zone"
-				onDrop={ this.onDrop }
-			/>
-		);
-	}
-}
+	};
+
+	return (
+		<DropZone
+			className="editor-block-drop-zone"
+			onDrop={ onDrop }
+		/>
+	);
+};
 
 BlockDropZone.propTypes = {
-	updateBlockAttributes: PropTypes.func,
 	srcClientId: PropTypes.string,
 	srcBlockName: PropTypes.string,
 };
 
-export default withDispatch( ( dispatch ) => {
-	const { updateBlockAttributes } = dispatch( 'core/block-editor' );
-
-	return {
-		updateBlockAttributes( ...args ) {
```

Comment:
This old code is really weird? Is there a purpose here?

Reply:
I don‘t think so. It was just more complicated than needed.